Configuration/Geometry/README.md

```diff
@@ -8,6 +8,8 @@ Tracker:
 * T14: Phase2 tilted tracker (v6.1.6) w/ phase 2 pixel (v6.1.3) (Based from T12. OT: reduced envelope. IT: new chip size, different radii, 2x2 modules everywhere in TEPX, new ring paradigm in TEPX)
 * T15: Phase2 tilted tracker (v6.1.6) w/ phase 2 pixel (v6.1.3) (Active geometry: same as T14. Material Budget: major update in IT, gathering info from recent Mechanical designs.)
 * T16: Active geometry: skewed Inner Tracker geometry. Material Budget: same as T15.
+* T17: Phase2 tilted tracker (v6.1.6) w/ phase 2 pixel (v6.1.5) TFPX: Changed sensors spacing within all double-disks + Increased distance between Disks 6 and 7 + Put TBPX portcards between Disks 6 and 7.
+* T18: Phase2 tilted tracker (v6.1.6) w/ phase 2 pixel (v6.1.6) TFPX: Shift all double-disks by + 25mm in Z. All services volumes (& TBPX portcards) are also shifted.
```

Comment:
has this been checked for overlaps?

Reply:
Yes i always solved all overlaps (it's a show-stopper for tracking) + did the other checks mentioned in PR validation.